tasks/post_to_catalog/schemas/catalog_record_input.json

```diff
@@ -55,6 +55,7 @@
       "type": "object",
       "description": "A dictionary containing granule information",
       "required": [
+        "providerId",
```

Comment:
I don not believe we want this here. If we do, we should get rid of the provider block above. as a result, we would also need to change copy_to_gralcier to work with the changes to the SQS schema here. I feel like since we have the provider block above, we are ok.

Reply:
removed providerId under granules in the schema